module/applications/item/ability-use-dialog.mjs

```diff
@@ -198,7 +200,12 @@ export default class AbilityUseDialog extends Dialog {
           const doc = profile.uuid ? fromUuidSync(profile.uuid) : null;
           const withinRange = ((profile.level.min ?? -Infinity) <= level) && (level <= (profile.level.max ?? Infinity));
           if ( !doc || !withinRange ) return null;
-          const label = profile.name ? profile.name : (doc?.name ?? "—");
+          let label = profile.name ? profile.name : (doc?.name ?? "—");
+          let count = simplifyRollFormula(Roll.replaceFormulaData(profile.count ?? "1", rollData));
+          if ( Number.isNumeric(count) ) {
+            count = parseInt(count);
+            if ( count > 1 ) label = `${count} x ${label}`;
```

Comment:
You think `3x Mummy` is more clear than `3 x Mummy`?

Reply:
That's usually how I see those sorts of quantities written, as `3x` without any space. Or the other way around as `x5` after the subject. I understand why we don't do this in the non-numeric case because `1d6x Ghoul` looks like an exploding d6, so `1d6 x Ghoul` is clearer. `1d6 Ghouls` is probably ideal but probably not achievable since we don't have pluralisation information for monster names anywhere.

Anyway, just a minor suggestion, the PR is approved so you can take it or leave it.